### PR TITLE
[BEAM-5969] Add BOUNDED_SIDE_INPUT_JOIN new Query to Beam website

### PR DESCRIPTION
**Pull request by echauchot**
Now that the new query is merged into master, let's add it on the website and also give the names of the queries. I also updated the status of that query in batch and streaming modes in the tables. As PCollectionView is not supported in spark in streaming mode, I put ko on that and ok on the others (I still need to check the status of that query by updating the dashboards)
R: @kennknowles 

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by echauchot** [comment]
@kennknowles quick update: I have just added the new query to all the perfkit dashboards for all the runners in batch and streaming modes. It runs fine !
Just a comment: nexmark postcommits on dataflow runner seem deactivated as there is no BQ table for that query. Nevertheless, I'll keep the dashboards for the time they are back online

**Comment by echauchot** [comment]
@kennknowles actually nexmark dataflow postCommits are not deactivated: they seem to be failing. I opened a ticket for that: https://issues.apache.org/jira/browse/BEAM-5996

File: website/src/documentation/sdks/nexmark.md [hunk 1]
```diff
@@ -171,18 +173,24 @@ Number of events generators:
...
     --numEventGenerators=4
 
-Run query N:
+Queries can be run by their name or by their number (number is still there for backward compatibility, only the queries 0 to 12 have a number)
+
+Run query **N**:
 
     --query=N
 
+Run query called **PASSTROUGH**:
```
Comment (kennknowles) [inline website/src/documentation/sdks/nexmark.md]: Typo: missing `H` and also below

File: website/src/documentation/sdks/nexmark.md [hunk 2]
```diff
@@ -171,18 +173,24 @@ Number of events generators:
...
+
 ### Available Suites
 The suite to run can be chosen using this configuration parameter:
 
     --suite=SUITE
 
 Available suites are:
 * DEFAULT: Test default configuration with query 0.
-* SMOKE: Run the 12 default configurations.
+* SMOKE: Run the 14 queries with the default configuration.
```
Comment (kennknowles) [inline website/src/documentation/sdks/nexmark.md]: Maybe just "all queries" to make it future-proof :-)

File: website/src/documentation/sdks/nexmark.md [hunk 3]
```diff
@@ -347,6 +355,13 @@ These tables contain statuses of the queries runs in the different runners. Goog
       <td>ok</td>
       <td>ok</td>
     </tr>
+    <tr>
+      <td>BOUNDED_SIDE_INPUT_JOIN</td>
```
Comment (kennknowles) [inline website/src/documentation/sdks/nexmark.md]: BTW from my reading of the bug the problem with Dataflow runner is not this query. So it is "OK" for all just like you have here.

File: website/src/documentation/sdks/nexmark.md [hunk 4]
```diff
@@ -679,7 +701,7 @@ There are dashboards for these runners (others to come):
 Each dashboard contains:
 - graphs in batch mode
 - graphs in streaming mode
-- graphs for the 13 queries.
+- graphs for the 14 queries.
```
Comment (kennknowles) [inline website/src/documentation/sdks/nexmark.md]: all queries?

File: website/src/documentation/sdks/nexmark.md (hunk 2 above)
Comment (echauchot) [reply website/src/documentation/sdks/nexmark.md]: sure :) thx

File: website/src/documentation/sdks/nexmark.md (hunk 3 above)
Comment (echauchot) [reply website/src/documentation/sdks/nexmark.md]: Good do know !

**Comment by echauchot** [comment]
@kennknowles I updated and squashed commits PTAL.

**Comment by echauchot** [comment]
@kennknowles ping ?

**Comment by kennknowles** [comment]
Sorry - I went on a little vacation without announcing. LGTM.

File: website/src/documentation/sdks/nexmark.md (hunk 1 above)
Comment (kennknowles) [reply website/src/documentation/sdks/nexmark.md]: Wait actually still missing the H

**Comment by kennknowles** [comment]
No need to wait for me again - merge when you are ready.

**Comment by echauchot** [comment]
no problem Kenn, thanks for the approval !

File: website/src/documentation/sdks/nexmark.md (hunk 1 above)
Comment (echauchot) [reply website/src/documentation/sdks/nexmark.md]: oups, this conversation was left resolved so I missed this comment. I'll fix in another PR then

**Pull request merged**